### PR TITLE
Browse by series

### PR DESCRIPTION
**Pull request by vladigor**
Added browse by series and refactored events_search to take a parameters dict instead of the whole request object.

### CODE REVIEW

File: talks/api/views.py [hunk 1]
```diff
@@ -110,12 +110,22 @@ def api_event_group_ics(request, event_group_slug):
 
 
 @api_view(["GET"])
+def suggest_event_group(request):
+    """Get event group titles for typeahead searching
+    """
+    query = request.GET.get('q', '')
+    series = EventGroup.objects.filter(Q(title__startswith=query)).distinct()
```
Comment (ahaith) [inline talks/api/views.py]: We might want to consider using a LIKE query here: (title__contains='query').
In an ideal world we'd index series titles into solr and search from there, but that's way beyond scope for now.

**Comment by ahaith** [comment]
Merging as this works nicely. We'll see if Anne comes back with any feedback regarding starts_with vs contains for series title matches.

**Comment by envycontent** [comment]
I might have missed this one. Can you send me a link and I'll pursue.

I'll set aside some time this morning and work through stuff.

Anne

**Comment by ahaith** [comment]
I hadn't previously mentioned it, but it's ready to look at now. We've just deployed to the dev server so it'd be good to get some feedback from you, including on how best to match series to partially typed strings.
N.B. series functionality currently sometimes yields errors due to the SSL certificate being out of date. The request is now in for a fresh one.

**Comment by envycontent** [comment]
Nicole and I are testing things out now. Nicole is going to start a
google doc and share with you - just so that we can feed back straight
away - I'm happy to turn these into issues on github in due course, but
have got to go to a meeting now and probably won't be back at my desk
until after 5pm.

Anne

**Pull request merged**